frontend/src/components/SortableTask.module.scss

```
@@ -34,3 +34,18 @@
     height: 100%;
   }
 }
+
+.taskDetailsTooltip {
+  @extend .layout-column;
+  align-items: center;
+  gap: 20px;
+
+  .title {
+    margin-bottom: 2px;
+  }
+
+  .customerValues {
+    @extend .layout-row;
```

```suggestion
    display: grid;
    grid-template-columns: repeat(3, auto);
```

The flex row overflows with some more customers.

Maybe could use a grid with some number of columns instead, allowing the content to flow to multiple rows.

---

Good point. Implemented with flex-wrap (it is easier to center to me)

---

That works as well, if not better 👍